templates/alert_form.html.twig

```diff
@@ -0,0 +1,252 @@
+        __("Template"),
+        {
+            'display_emptychoice': true,
+        }|merge(base_option_small)
+    ) }}
+    {{ fields.nullField() }}
+
+    {{ fields.dropdownArrayField(
+        'size',
+        item.fields.size,
+        sizes,
+        __("Size"),
+        base_option_small
+    ) }}
+    {{ news_fields.iconField(
+        "icon",
+        icons,
+        item.fields.icon,
+        __('Icon'),
+        {
+            'display_emptychoice': true,
+        }|merge(base_option_small)
+    ) }}
+    {{ fields.dropdownYesNo(
+        'display_dates',
+        item.fields.display_dates,
+        __('Show dates'),
+        base_option_small
+    ) }}
+
+    {{ news_fields.colorfield(
+        "background_color",
+        colors,
+        item.fields.background_color,
+        __('Background color'),
+        base_option_small
+    )}}
+    {{ news_fields.colorfield(
+        "text_color",
+        colors,
+        item.fields.text_color,
+        __('Text color'),
+        base_option_small
+    )}}
+    {{ news_fields.colorfield(
+        "accent_color",
+        colors,
+        item.fields.accent_color,
+        __('Accent color'),
+        base_option_small
+    )}}
+
+    <div class="hr-text preview-section {{ item.isNewItem() ? "d-none" : "" }}">
+        <i class="ti ti-eye"></i>
+        <span>{{ __('Preview') }}</span>
+    </div>
+
+    <div class="alert-preview">
+    </div>
+
+    <script>
+        (async function () {
+            const reload_preview = function () {
+                // Show preview section if hidden
+                $('.preview-section').removeClass('d-none');
+
+                // Send tinymce content to the forms textarea
+                tinyMCE.triggerSave();
+
+                // Load preview
+                $(".alert-preview").load("{{ preview_url }}" + "?" + $("form[name=asset_form]").serialize());
+            };
+
+            // Debounced function to not spam the backend for changes less than 0.4 seconds appart
+            const reload_preview_debounced = _.debounce(reload_preview, 400, false);
+
+            // Any input or dropdown change on our form
+            let updating_template = false;
+            $("form[name=asset_form] :input").on('input', function(e) {
+                if (updating_template) {
+                    // Prevent infinite loops as modifying the template will trigger
+                    // an update on others fields
+                    return;
+                }
+
+                if ($(e.target).prop('name') == 'type') {
+                    // Template change
+                    updating_template = true;
+
+                    // Template disabled, stop processing
+                    if ($(e.target).val() == "0") {
+                        updating_template = false;
+                        return;
+                    }
+
+                    // Clear values
+                    $("form[name=asset_form] :input[name=icon]").prop('checked', false);
```

Comment:
Why an extra `:`?
```suggestion
                    $("form[name=asset_form] input[name=icon]").prop('checked', false);
```

Reply:
I've used it in some places to target both `input` and `select` items.
I guess it isn't need here tho, done 9cce621.